src/index.html

```diff
@@ -11,7 +11,170 @@
+
+        <label name="surname">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          required
+          >
+        </label>
+
+        <label name="name">
+          Name:
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          required
+          >
+        </label>
+
+        <label name="age">
+          How old are You?
+          <input
+          type="number"
+          name="age"
+          required
+          min="1"
+          max="100"
+          value="12"
+          >
+        </label>
+
+        <label name="date-of-birth">
+          Full date of birth:
+          <input
+          type="date"
+          name="date-of-birth"
+          required
+          >
+        </label>
+
+        <label name="terms">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="terms"
+          required
+          >
+        </label>
+      </fieldset>
+      
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+
+        <label name="email">
+          E-mail:
+          <input
+          type="email"
+          placeholder="email@example.com"
+          required
+          >
+        </label>
+        
+        <label name="password">
+          Password:
+          <input
+          type="password"
+          required
+          minlength="8"
+          maxlength="32"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-field" name="cat-love">
+        <legend>An interesting fact about you!</legend>
+          Do you love cats?
+        <label class="question-radio">
+          <input
+            type="radio"
+            name="question"
+            value="yes"
+          >
+            Yes
+        </label>
+  
+        <label class="question-radio">
+          <input
+            type="radio"
+            name="question"
+            value="no"
+          >
+            No
+        </label>
+        
+        <label name="favorite-color">
+          What is your favorite color?
+          <input type="color">
+        </label>
+
+        <label name="sleep-time">
+          What time do you go to bed?
+          <input type="time">
+        </label>
+
+        <label name="favorite-car">
+          What are your favorite brand of cars?
+          <select
+          name="cars"
+          id="cars"
+          multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label name="mark">
+          How do you rate our work?
+          <input type="range">
+        </label>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>Additional info:</legend>
+
+        <label name="comments">
+          Comments:
```

Comment:
checklist p.8  Check the textarea behaviour - when you click on the textarea, the coursor should appear on the beginning of the input.